task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +528,33 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let gain; 
+
+    let verif = position.some((value, i, arr) => {     
```

Comment:
If possible you should pass pure functions to Array.prototype.* functions like reduce, map, some etc.

This function you pass to **some** method changes variable outside of its scope ( **gain** )  which makes it not a pure function.

Try to refactor this to use pure funciton. Try to use find method to find filled row

https://www.tutorialspoint.com/what-is-a-pure-function-in-javascript

Reply:
It was slow but I hope my solution looks good now